src/cpp/configuration/implementations/Address_configuration.cpp

```diff
@@ -48,6 +119,24 @@ Address::Address(
         ip_ = Address::default_ip();
     }
 
+    // Check the IP Version is correctly set
```

Comment:
Remove so checks are also performed for the case when ip_version is provided.

Reply:
Although just removing the conditional would be not enough, as ip_version could change from the one provided without the user noticing.

Reply:
old version